client-py/SessionExample.py

```diff
@@ -178,19 +185,41 @@
 
 # insert one numpy tablet into the database.
 np_values_ = [
-    np.array([False, True, False, True], np.dtype('>?')),
-    np.array([10, 100, 100, 0], np.dtype('>i4')),
-    np.array([11, 11111, 1, 0], np.dtype('>i8')),
-    np.array([1.1, 1.25, 188.1, 0], np.dtype('>f4')),
-    np.array([10011.1, 101.0, 688.25, 6.25], np.dtype('>f8')),
+    np.array([False, True, False, True], np.dtype(">?")),
+    np.array([10, 100, 100, 0], np.dtype(">i4")),
+    np.array([11, 11111, 1, 0], np.dtype(">i8")),
+    np.array([1.1, 1.25, 188.1, 0], np.dtype(">f4")),
+    np.array([10011.1, 101.0, 688.25, 6.25], np.dtype(">f8")),
     np.array(["test01", "test02", "test03", "test04"]),
```

Comment:
Hi, would you like to format the example code in user guide documents?
`client-py/README.md`, `docs/UserGuide/API/Programming-Python-Native-API.md`and `docs/zh/UserGuide/API/Programming-Python-Native-API.md`.

Reply:
I've formatted the example code in docs.